tests/console/check_resume.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify that "resume=" kernel parameter is absent in the list of default parameters on Sle15-SP2
+# see https://jira.suse.com/browse/SLE-6926
+
+# Maintainer: Jonathan Rivrain <jrivrain@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    if (check_var('ARCH', 's390x')) {
```

Comment:
What's about using `is_s390x` from `lib/Utils/Architectures.pm`?

Reply:
I removed this test, as we'll scedule it only on s390 anyway. Is that ok ?

Reply:
Yes, I guess it's even better, as we won't pretend that test worked anywhere else if is accidentally scheduled.